README.md

```diff
@@ -57,36 +60,63 @@ Then re-run:
 yarn backend
 
 ```
+![Screen Shot 2021-11-20 at 9 51 20 AM](https://user-images.githubusercontent.com/9419140/142730794-cdaad88f-9ce4-4cb9-af61-7814b135e1db.png)
 
 ---
 
 ## Usage
 
 📱 Open <http://localhost:3000> to see the app
 
-![image](https://user-images.githubusercontent.com/2156509/135258832-61bcc08f-68be-4cb8-9493-15a4e0e6be98.png)
+![Screen Shot 2021-11-20 at 9 29 18 AM](https://user-images.githubusercontent.com/9419140/142730015-31d4b18c-8f4d-406f-a8da-84ba6742e308.png)
+
+Connect your ethereum wallet by clicking "Connect Wallet" 
+
+You need to register as a buidler next.
+
+![Screen Shot 2021-11-20 at 9 29 00 AM](https://user-images.githubusercontent.com/9419140/142730187-75941ef8-15b3-406d-ab6b-e175c21e522f.png)
+
+![Screen Shot 2021-11-20 at 9 37 36 AM](https://user-images.githubusercontent.com/9419140/142730302-63a60016-b7cb-4e69-a39b-93aa7a24f30e.png)
+
+You will now see your home page after signing a message.
+
+![Screen Shot 2021-11-20 at 9 36 31 AM](https://user-images.githubusercontent.com/9419140/142730252-a45af7af-98fd-4fd8-b639-4c478743163e.png)
+
+You can view all buidler by selecting "Buidlers"
```

Comment:
@codenamejason Newly created users can only see "Portfolio" in their menu. Builders / Builds / Review are only for builders / admins.

We might want to explain the roles on a earlier section showing this screens (anon, registered, builder, admin). I can do that :D